recipe-server/client/control/components/recipes/RecipeDetails.js

```diff
@@ -30,6 +30,50 @@ export default class RecipeDetails extends React.PureComponent {
     }
   }
 
+  renderArgumentValue(name, value) {
+    switch (name) {
```

Comment:
I think these `switch`es could be simplified down to a couple `if`s. It's a little weird to read at first since there are two `switch`es in a row with only one `case` each.

It may be better without the big block of JSX, too - I'd suggest making the `<table>` its own component so reading through this function is a bit more concise.

Reply:
Yeah, the switches are overkill. I was imagining having more cases, and then never simplified after I realized I only had one case in each.